GitOut/Features/Git/LocalGitRepository.cs

```diff
@@ -385,6 +385,8 @@ public async Task ExecuteCheckoutBranchAsync(GitBranchName name)
             }
         }
 
+        public Task ExecuteIntentToAdd(GitStatusChange change) => CreateProcess(ProcessOptions.FromArguments($"add --intent-to-add -- {change.Path}")).ExecuteAsync();
```

Comment:
Remove "Execute" as talked about earlier

Reply:
I think should be placed next to `AddAsync` (and renamed to `AddWithIntentAsync`).
Could this instead be added as an option to the `AddAsync` method, `Repository.AddAsync(change, AddOptions.WithIntent())`?